src/object-properties.js

```diff
@@ -1,38 +1,62 @@
 import DEFAULTS from './style-defaults';
 
+/**
+ * @extends {qae.GenericObjectProperties}
+ * @entry
+ */
+
 const properties = {
   /**
-   * @type {boolean}
+   * Current version of this generic object definition.
+   * @type {string}
+   * @default
    */
-  showTitles: false,
+  version: (process && process.env && process.env.PACKAGE_VERSION) || '1.0.0',
```

Comment:
When is the version nbr not available?

Reply:
When running rendering tests, but it is still failing. process comes from node it seems, I'll have to get around it in a different way